docs/diginstroom/sip/1.0/sip_structure/6_structure_representation.md

```diff
@@ -231,6 +259,63 @@ Since the `dmdSec`, `amdSec`, `fileSec` and `structMap` sections follow the same
 | Cardinality | 0..1 |
 | Obligation | MAY |
 
+### \<structMap\> section
+
+The `structMap` element outlines the hierarchical structure of the representation level of the SIP.
+Its requirements are very similar to the package level, however, instead of pointing to the contained representations, a `Data` division points to the contained files. 
```

Comment:
E-ARK CSIP heeft zelf de groepsnamen gedefinieerd die gebruikt mogen worden in de structMap: https://earkcsip.dilcis.eu/schema/CSIPVocabularyFileGrpAndStructMapDivisionLabel.xml. `Data` zit daar niet tussen (maar je mag wel uitbreiden met een extern vocabularium). Helaas vind ik geen (up-to-date) voorbeelden van een Representatie METS van E-ARK. Misschien dat daarvoor die `@LABEL='Representations"` ([CSIP101](https://earkcsip.dilcis.eu/#CSIP101)) dient? 

In die CSIPVocabularyFileGrpAndStructMapDivisionLabel.xml staat bijvoorbeeld bij 'Representations': _The file group or structural division contains data or representation data_

wij hebben het ook opgelost met een 'Data' division.